insta_reels_publishing_api_sample/upload_page.pug

```diff
@@ -12,16 +12,16 @@ html
         header
             h1 Instagram Reels Publishing API Sample App
         .canvas
-            - let pagesArr = pages ?? [];
-            if (pagesArr.length > 0)
+            - let accountsArr = accounts ?? [];
+            if (accountsArr.length > 0)
                 form(action='/uploadReels' name="uploadForm" id="uploadForm" method='POST')
-                    b Select an Page associated with Instagram Business Account ID to publish reel to
+                    b Select an Instagram Business Account to publish reel to
                     div(class='radio-group')
-                        each page, index in pagesArr
+                        each account, index in accountsArr
                             div(class='radio-child')
-                                label(for=`page-${index}`)
-                                    input(type='radio' id=`page-${index}` name="pageID" value=page.id)
-                                    |  Page: #{page.name} - #{page.id}
+                                label(for=`account-${index}`)
+                                    input(type='radio' id=`account-${index}` name="accountId" value=account.id)
+                                    |  Account: @#{account.username} (#{account.name}) - #{account.id}
```

Comment:
If account.name is null, its displaying something like `@instaaccountxyz () -- 123456`. I think we can make that part conditional so as to avoid getting the empty parantheses @pestevez

Reply:
Ah, good catch! I was under the assumption that name was a required field. I moved the logic to generate a `displayName` property to the index file.